src/EventService.Data/CategoryRepository.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using LT.DigitalOffice.EventService.Data.Interfaces;
+using LT.DigitalOffice.EventService.Data.Provider;
+using Microsoft.EntityFrameworkCore;
+
+namespace LT.DigitalOffice.EventService.Data;
+
+public class CategoryRepository : ICategoryRepository
+{
+  private readonly IDataProvider _provider;
+  public async Task<bool> DoesExistAsync(List<Guid> categoryIds)
```

Comment:
Hmm, name is not clear. Reading this name I cannot get that can pass List of categories. My suggestions:
* Use params
* or Rename method to DoesExistAllAsync e.g., or smth like this